lib/ReactViews/Search/SidebarSearch.jsx

```diff
@@ -19,7 +20,9 @@ const SidebarSearch = observer(
     },
 
     backToNowViewing() {
```

Comment:
Both are equivalent, but it might be prettier to use the `@action` decorator here which you can import from `mobx-react` just like `@observer`.